src/javaRosa.js

```diff
@@ -560,10 +560,11 @@ define([
         block.refreshMessages = function () {
             // TODO improve this to display each message beside the
             // form and language to which it applies
```

Comment:
Since "each language has it's own error container" is this comment now obsolete?